class.jetpack.php

```diff
@@ -1485,12 +1493,8 @@ public static function get_active_plan() {
 		);
 
 		if ( in_array( $plan['product_slug'], $premium_plans ) ) {
-			$plan['supports'] = array(
-				'videopress',
-				'akismet',
-				'vaultpress',
-				'wordads',
-			);
```

Comment:
Did you mean to remove videopress and wordads from here?

Reply:
Yes - they are modules, the others are plugins. Modules are automatically added to the list based on metadata.